selection_tools/run_prepare_histograms.sub

```diff
@@ -1,7 +1,7 @@
 Universe              = vanilla
 executable            = run_prepare_histograms.sh
-#			process			part
-arguments             = tta_mAlp-2GeV 	$(ProcId)
+#			process			part	    username
+arguments             = tta_mAlp-10GeV 	$(ProcId)   jniedzie
```

Comment:
Same as above - no need to pass username if we use "whoami" (unless it didn't work for you for some reason - then we should fix it properly).

Reply:
The username is for the input path, since the generated samples were either in yours or mine dust space. Now they are all in yours again so I need your username as input. I can clarify this in the code maybe.
But it's good for the other paths that you've commented on so I can add it there!

Reply:
Well, my point in general is that we don't need to specify username by hand anywhere - you can always get it programmatically.